Dockerfile

```diff
@@ -39,23 +35,22 @@ RUN apt-get -qq update && \
     wget http://packages.osrfoundation.org/gazebo.key -O - | apt-key add - && \
     add-apt-repository ppa:kisak/kisak-mesa && \
     curl -sSL http://get.gazebosim.org | sh && \
-    update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-10 800 --slave /usr/bin/g++ g++ /usr/bin/g++-10 --slave /usr/bin/gcov gcov /usr/bin/gcov-10 && \
```

Comment:
we already have 11th version of compiler in image

Reply:
hm... it's not linking with g++10/11, it asks -fPIC, strange... Should investigate it, but later

Reply:
about linking: https://github.com/gazebosim/gz-rendering/pull/950